library/Reporting/Database.php

```diff
@@ -16,7 +16,7 @@ protected function getDb(): RetryConnection
     {
         $config = new Sql\Config(
             ResourceFactory::getResourceConfig(
-                Config::module('reporting')->get('backend', 'resource', 'reporting')
+                Config::module('reporting', 'config', true)->get('backend', 'resource', 'reporting')
```

Comment:
Okay, but doesn't this apply also to web requests? How often is a connection attempt made there? How often would that read from disk with no reason?